superset/assets/javascripts/modules/superset.js

```diff
@@ -0,0 +1,293 @@
+/* eslint camel-case: 0 */
```

Comment:
I think this is a merge conflict, this file has been deleted as part of the Dashboard refactor, let's not re-introduce this file